packages/nextjs/app/my-grants/page.tsx

```diff
@@ -51,32 +52,41 @@ const MyGrants: NextPage = () => {
             </div>
           )}
           <p className="m-0">{grant.description}</p>
-          <p className={`badge ${badgeBgColor[grant.status]}`}>{grant.status}</p>
-          {grant.approvedTx && (
-            <a
-              href={getBlockExplorerTxLink(Number(grant.txChainId), grant.approvedTx)}
-              target="_blank"
-              rel="noopener noreferrer"
-              className="ml-4 underline underline-offset-4 text-xs"
-            >
-              50% approve tx <ArrowTopRightOnSquareIcon className="h-4 w-4 inline" />
-            </a>
-          )}
-          {grant.completedTx && (
-            <a
-              href={getBlockExplorerTxLink(Number(grant.txChainId), grant.completedTx)}
-              target="_blank"
-              rel="noopener noreferrer"
-              className="ml-4 underline underline-offset-4 text-xs"
-            >
-              50% complete tx <ArrowTopRightOnSquareIcon className="h-4 w-4 inline" />
-            </a>
-          )}
-          {grant.status === PROPOSAL_STATUS.APPROVED && (
-            <button onClick={() => openModal(grant)} className="btn btn-primary float-right">
-              Submit build
-            </button>
-          )}
+          <div className="flex items-center justify-between">
+            <div className="flex items-center">
+              <p className={`badge ${badgeBgColor[grant.status]}`}>{grant.status}</p>
+              {grant.note && grant.note.trim().length > 0 && (
```

Comment:
Another nitpick, in this case in the logic, maybe we'll need to check this with Carlos/Austin.

We don't have notes in the Completion step, but at /my-grants we still show Notes from the grant (if they have a note in database, which will be the notes from the first approval).

- Should we hide Notes (?) Tooltip in /my-grants for completed status
- Should we add a Notes field when completing a grant (probably not!)
- Or should we just do nothing? (maybe in case first approval notes are going to be some kind of motivational or ever-green feedback)

Reply:
Option 1 is what I had in mind when writing the spec! 

Fixed

a01c909a140a9831df58698c909c0c5e0e928af9

Thanks, good catch!